src/orchestrator/streaming.rs

```diff
@@ -271,11 +271,17 @@ async fn streaming_output_detection_task(
         // Create a mutable copy of the parameters, so that we can modify it based on processing
         let mut detector_params = detector_params.clone();
         let detector_id = detector_id.to_string();
-        let chunker_id = ctx.config.get_chunker_id(&detector_id).unwrap();
+        let chunker_id = ctx.config.get_chunker_id(&detector_id).expect(&format!(
+            "error getting chunker id from detector: {}",
+            detector_id
+        ));
 
         // Get the detector config
         // TODO: Add error handling
-        let detector_config = ctx.config.detectors.get(&detector_id).unwrap();
+        let detector_config = ctx.config.detectors.get(&detector_id).expect(&format!(
+            "error getting detector config with detector: {}",
+            detector_id
+        ));
```

Comment:
If a client requests an invalid `detector_id`, we should return an error to the user rather than panic and terminate the task. I can't recall if we already check this before processing, but the comment above leads me to believe that we don't. Need to review further.